src/plugins/simulator/fs9/simulatorfs9.cpp

```diff
@@ -416,22 +441,23 @@ namespace BlackSimPlugin::Fs9
         // To avoid jitters, I wait some update cylces to stabilize the values
         if (m_skipCockpitUpdateCycles < 1)
         {
-            const auto dontOverwrite = [](const CFrequency &from, const CFrequency &to) {
-                // don't overwrite 8.33 kHz frequencies
-                return CComSystem::isExclusiveWithin8_33kHzChannel(from) &&
-                       from.valueInteger(CFrequencyUnit::kHz()) / 10 == to.valueInteger(CFrequencyUnit::kHz()) / 10;
-            };
+            // const auto dontOverwrite = [](const CFrequency &from, const CFrequency &to) {
+            //     // don't overwrite 8.33 kHz frequencies
+            //     return CComSystem::isExclusiveWithin8_33kHzChannel(from);
+            //     // && from.valueInteger(CFrequencyUnit::kHz()) / 10 == to.valueInteger(CFrequencyUnit::kHz()) / 10;
+            // };
+
             CComSystem com1 = simDataOwnAircraft.getCom1System();
             CComSystem com2 = simDataOwnAircraft.getCom2System();
             const CComSystem oldCom1 = getOwnComSystem(CComSystem::Com1);
             const CComSystem oldCom2 = getOwnComSystem(CComSystem::Com2);
-            if (dontOverwrite(oldCom1.getFrequencyActive(), com1.getFrequencyActive()))
+            if (CSimulatorFs9::dontOverwrite(oldCom1.getFrequencyActive(), com1.getFrequencyActive()))
                 com1.setFrequencyActive(oldCom1.getFrequencyActive());
-            if (dontOverwrite(oldCom1.getFrequencyStandby(), com1.getFrequencyStandby()))
+            if (CSimulatorFs9::dontOverwrite(oldCom1.getFrequencyStandby(), com1.getFrequencyStandby()))
                 com1.setFrequencyStandby(oldCom1.getFrequencyStandby());
-            if (dontOverwrite(oldCom2.getFrequencyActive(), com2.getFrequencyActive()))
+            if (CSimulatorFs9::dontOverwrite(oldCom2.getFrequencyActive(), com2.getFrequencyActive()))
                 com2.setFrequencyActive(oldCom2.getFrequencyActive());
-            if (dontOverwrite(oldCom2.getFrequencyStandby(), com2.getFrequencyStandby()))
+            if (CSimulatorFs9::dontOverwrite(oldCom2.getFrequencyStandby(), com2.getFrequencyStandby()))
```

Comment:
With this, can the user switch away from a 8.33 kHz frequency by only using the simulator? ``dontOverwrite`` only checks the first parameter if it is a 8.33 kHz frequency. Hence, if oldCom is a 8.33 kHz frequency tuned from within swift, all changes from the simulator will always be skipped.

Reply:
yes, that's exactly how it's intended. Only when 25kHz frequencies are set again in swift will the values be adopted by the simulator. This is part of the solution, otherwise the simulator will overwrite the 8.33 frequency with incorrect values.

Reply:
Perhaps it would be a good idea to document the behavior of the non-8.33 kHz capable simulators in the user documentation. I will do this soon.